layouts/black-lives-matter.hbs

```diff
@@ -220,7 +220,7 @@
       <li>William Green</li>
       <li>Willie McCoy</li>
       <li>Yassin Mohamed</li>
-      <li>... <a href="https://mappingpoliceviolence.org/unarmed">and</a> <a href="https://mappingpoliceviolence.org/unarmed2014">countless</a> <a href="https://apps.npr.org/documents/document.html?id=6933593-NPR-CodeSwitch-Saytheirnameslistv3">more</a> <a href="https://ebwiki.org/">names</a> not listed here whose lives were taken by violence and brutality.</li>
+      <li>&hellip;<a href="https://mappingpoliceviolence.org/unarmed">and</a> <a href="https://mappingpoliceviolence.org/unarmed2014">countless</a> <a href="https://apps.npr.org/documents/document.html?id=6933593-NPR-CodeSwitch-Saytheirnameslistv3">more</a> <a href="https://ebwiki.org/">lives</a> taken by violence and brutality.</li>
```

Comment:
```suggestions
      <li>&hellip;<a href="https://mappingpoliceviolence.org/unarmed">and</a> <a href="https://mappingpoliceviolence.org/unarmed2014">countless</a> <a href="https://apps.npr.org/documents/document.html?id=6933593-NPR-CodeSwitch-Saytheirnameslistv3">more</a> <a href="https://ebwiki.org/">lives</a> not listed here, taken by violence and brutality.</li>

```

Reply:
re-adding the "not listed here" part since that was something that was added with intention

Reply:
What was the intention? "and countless other lives" is inherently "not listed here".

Reply:
"and countless other lives" asserts that there are more more Black lives that have been taken, and _implies_ that we're not listing them. Given that this is in-line with Say Their Names, it feels more appropriate to explicitly assert that we're _not_ saying more names.

Reply:
Thanks for the explanation.